3542/1/lib/repo.rb

```diff
@@ -0,0 +1,26 @@
+require 'httparty'
+require 'nokogiri'
+require 'open-uri'
+require_relative 'util'
+require_relative 'parser'
+
+class Repo
+  attr_reader :info
+
+  def initialize(gem_name)
+    parser = Parser.new gem_name
```

Comment:
I would not mention any classes here.
In you case class Repo depends on class Parser.
```
info = Parser.new(gem_name)
repo = Repo.new(info)

class Repo
 def initialize(info)
   @info = info
 end
end
```